src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -158,7 +180,13 @@ export default defineComponent({
     const elPlay = () => {
       if (window?.audioEl) {
         if (window.audioEl.src !== props.audio.url) {
-          window.audioEl.src = props.audio.url // resets position to zero
+          window.audioEl.src = props.audio.url
+          // Set the current time of the audio back to the seeked time
+          // of the waveform/timeline. In the future we might change
+          // this to reset the audio back to 0 anytime another audio
+          // is played but for now this is simpler and requires less
+          // cross-instance communication.
+          window.audioEl.currentTime = currentTime.value
```

Comment:
```suggestion
          // Set the current time of the audio back to the seeked time
          // of the waveform/timeline. In the future we might change
          // this to reset the audio back to 0 anytime another audio
          // is played but for now this is simpler and requires less
          // cross-instance communication.
          window.audioEl.currentTime = currentTime.value
          window.audioEl.src = props.audio.url
```

@sarayourfriend this one weird trick fixes all of the problems for me! I haven't tested this + _removing_ your other attempts to fix browser bugs but I will do so now.